plugins/migration/bin/command.js

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env node
```

Comment:
Unless I'm mistaken, migrate.js is the only file in this package that's actually an executable script in its own right (i.e., should be +x and have a `#!` line); the rest are just regular JS files.  (Here and most files below.)